app-android-xamarin/Moodis/Moodis/Resources/layout/activity_menu.xml

```diff
@@ -0,0 +1,173 @@
+﻿
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ImageView
+        android:id="@+id/imageForView"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/grid_unit_190x"
```

Comment:
use `0dp` instead, so the height would be as big as constraint layout would let(without overlapping with other components). Also, avoid concrete values as much as possible when dealing with component sizes, because if you use concrete values some components would not look the way it should on different screen sizes.

Reply:
And put bottom to top constraint, so the `0dp` value would work